server/src/services/organisation.js

```diff
@@ -177,6 +178,28 @@ const hasUser = async (id, userId, knex) => {
     return hasUser.length > 0
 }
 
+const generateInvitationToken = ({ organisation, user, secret }, emailTo) => {
+    secret = process.env.INTERNAL_JWT_SECRET
+    if (!secret) {
+        throw new Error('No secret setup, set INTERNAL_JWT_SECRET')
+    }
```

Comment:
Since we want errors ASAP instead of at runtime, switching to this + `getRequiredEnv(process.env.INTERNAL_JWT_SECRET)` should make our lives easier:
```suggestion
const generateInvitationToken = ({ organisation, user }, emailTo) => {
    const secret = process.env.INTERNAL_JWT_SECRET
```